### PR TITLE
Add meeshkan tool.

### PR DESCRIPTION
**Pull request by ksaaskil**
Internal PR to our fork, after merge I'll make PR to upstream repo.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -432,6 +432,17 @@
   v2: false
   v3: true
 
+- name: meeshkan
```
Comment (carolstran) [inline _data/tools.yml]: Are we doing it like `meeshkan` = product, `Meeshkan` = company? That hasn't been clear to me so far.

File: _data/tools.yml [hunk 2]
```diff
@@ -432,6 +432,17 @@
...
+  category:
+    - learning
+    - mock
+  language: Python
+  link: https://github.com/meeshkan/meeshkan
+  github: https://github.com/meeshkan/meeshkan
+  description: Tool that mocks HTTP APIs for use in sandboxes as well as for automated and exploratory testing. It uses a combination of API definitions, recorded traffic and code in order to make crafting mocks as enjoyable as possible.
```
Comment (carolstran) [inline _data/tools.yml]:
```suggestion
  description: Mock HTTP APIs through a combination of API definitions, recorded traffic and code. Used for sandboxes, as well as automated and exploratory testing.
```

Trying to cut down the description as the others on the site are pretty short/snappy 🤔  wdyt?

**Review by carolstran — COMMENTED** [review]
Left two comments which make me hesitant to go one way or another (approving/requesting changes). Curious to hear what you think!

File: _data/tools.yml (hunk 1 above)
Comment (ksaaskil) [reply _data/tools.yml]: That's how I've understood it! It seems less commercial when you write in lowercase :D

File: _data/tools.yml (hunk 2 above)
Comment (ksaaskil) [reply _data/tools.yml]: Thanks, it's much better now and doesn't read like an ad.

**Review by carolstran — APPROVED** [review]
As discussed on Teams, I think it's fine as-is! I do wonder if we could iterate later once we refine our selling points, but that shouldn't block this PR. My gripe is just that we should capitalize `Meeshkan` because that's how we currently have it in the docs 😆

**Pull request merged**